simplq/src/components/pages/Admin/Token.jsx

```diff
@@ -98,7 +98,7 @@ function Token(props) {
           <p>{name}</p>
         </div>
         <div className={styles['token-operations']}>
-          <div className={styles['token-call-notification']}>
+          <div>
```

Comment:
can you remove this div altogether, it will fix the icons coming one below the other. Also add margins between tokens

![image](https://user-images.githubusercontent.com/11256376/94957869-2c8b3880-050c-11eb-9497-0e91383f97e3.png)

Reply:
It is not due to that div. It does not occur if the max-width is set to 40rem
![Screenshot 2020-10-03 100618](https://user-images.githubusercontent.com/59063979/94983442-7271ec00-0560-11eb-9270-a753c563320e.png)